pkg/agent/agent.go

```diff
@@ -431,6 +434,12 @@ func (i *Initializer) Initialize() error {
 		}
 	}
 
+	if i.enableL7Visibility {
+		if err := i.prepareL7VisibilityInterfaces(); err != nil {
```

Comment:
just want to confirm that it's ok to call both `i.prepareL7VisibilityInterfaces()` and `i.prepareL7NetworkPolicyInterfaces()` (if both `i.enableL7NetworkPolicy` and `i.enableL7Visibility` are true).

cc @tnqn 

I was expecting the code to be more like this:
```go
if i.enableL7NetworkPolicy || i.enableL7Visibility {
        i.prepareL7Interfaces()
}
```
with a shared setup for L7NP and L7Visibility if both are enabled

Reply:
I agree, since Suricata uses both the interfaces, we can create both interfaces if any of the featuregate (L7NP, L7Visibility)